browser-tests/functional-tests/functional.spec.js

```diff
@@ -69,6 +69,71 @@ test.describe.parallel("Functional tests", () => {
     const url = page.url();
     expect(url).toBe(`${domainUrl}/ipv/page/page-multiple-doc-check`);
   });
+
+  test("Successfully gets proven user details from core-back for the page-ipv-reuse screen", async ({ page }) => {
```

Comment:
I wasn't sure what other scenarios to include in the functional tests since things like handling of multiple names and address, throwing of errors are covered by the snapshot tests and/or unit tests

Reply:
Yep, I think what we're really looking for on reuse is to ensure that the user data is passed back correctly from the backend API request.

As you say, the snapshot tests cover the precise rendering behaviour, and the UTs cover the transformation logic.